bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,837 +4,560 @@
-		<label>Date Min CO2 This Week</label>
-		<description>Date when minimum CO2 was reached this week</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMinCo2ThisMonth" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Min CO2 This Month</label>
-		<description>Date when minimum CO2 was reached this month</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMaxCo2" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Max CO2</label>
-		<description>Date when maximum CO2 was reached on current day</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMaxCo2ThisWeek" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Max CO2 This Week</label>
-		<description>Date when maximum CO2 was reached this week</description>
-		<category>Time</category>
-		<state readOnly="true"/>
+		<label>Monitoring</label>
+		<description>Monitoring state of the camera</description>
 	</channel-type>
 
-	<channel-type id="dateMaxCo2ThisMonth" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Max CO2 This Month</label>
-		<description>Date when maximum CO2 was reached this month</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMinTemp" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Min Temp</label>
-		<description>Date when minimum temperature was reached on current day</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMinTempThisWeek" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Min Temp This Week</label>
-		<description>Date when minimum temperature was reached this week</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMinTempThisMonth" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Min Temp This Month</label>
-		<description>Date when minimum temperature was reached this month</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMaxTemp" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Max Temp</label>
-		<description>Date when maximum temperature was reached on current day</description>
-		<category>Time</category>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="dateMaxTempThisWeek" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Max Temp This Week</label>
-		<description>Date when maximum temperature was reached this week</description>
-		<category>Time</category>
+	<channel-type id="window-open">
+		<item-type>Switch</item-type>
+		<label>Window Status</label>
+		<description>Windows of the room are opened.</description>
```

Comment:
This is typically the case of channel that should be linked to a Contact item and not a Switch item.

Reply:
You're right. Anticipated-heating should also be a Contact.

Reply:
Enhanced.